magento/pwa-studio
# #2407 Add yarn serve

## Description
* Start a test server `yarn venia start`
* Start a production server `NODE_ENV=production yarn venia start`

### Preconditions
`pwa-studio/packages/venia-concept/.env` need to exits
`yarn build` need to be executed 


## Related Issue





Closes #2407

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps
1.  Start a test server should work
1.1. Forgetting run build should throw an error
2.  yarn run watch:venia should work

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
